src/backend/aspen/app/views/tests/test_phylo_tree_view.py

```diff
@@ -0,0 +1,53 @@
+import json
+import random
+
+from aspen.app.views.phylo_trees import PHYLO_TREE_KEY
+from aspen.test_infra.models.phylo_tree import phylorun_factory, phylotree_factory
+from aspen.test_infra.models.sample import sample_factory
+from aspen.test_infra.models.sequences import uploaded_pathogen_genome_factory
+from aspen.test_infra.models.usergroup import group_factory, user_factory
+
+
+def test_phylo_tree_view(
+    session,
+    app,
+    client,
+    n_samples=5,
+    n_trees=3,
+):
+    group = group_factory()
+    user = user_factory(group)
+    samples = [
+        sample_factory(
+            group,
+            public_identifier=f"public_identifier_%{ix}",
+            private_identifier=f"private_identifier_%{ix}",
```

Comment:
are you mixing up two different types of string formatting here?

Reply:
Yes, there should just be no `%`

Reply:
good catch!